drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ModelSynchronizerImpl.java

```diff
@@ -403,6 +403,26 @@ public void moveColumnsTo(final int targetColumnIndex,
         handler.moveColumnsTo(metaData);
     }
 
+    @Override
+    public void sort(final List<Integer> sortOrder) throws VetoException {
+
+        final List<Synchronizer> handlers = new ArrayList<>();
+        for (Synchronizer synchronizer : synchronizers) {
+            if (synchronizer.handlesSort()) {
+                handlers.add(synchronizer);
+            }
+        }
+
+        if (handlers.isEmpty()) {
+            throw new MoveVetoException();
+        }
```

Comment:
To be honest I followed this style since it was used in other methods. The benefit of this seems to be that the code below is not called.

```
public void updateSystemControlledColumnValues() {
        systemControlledColumnValuesSynchronizer.updateSystemControlledColumnValues();
}
```

Reply:
ok, thank you, keep it as it is